devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -131,7 +142,7 @@ def ping() {
 }
 
 def refresh() {
-	return zigbee.readAttribute(0x0006, 0x0000, [destEndpoint: 0xFF])
+	zigbee.onOffRefresh() + zigbee.readAttribute(zigbee.ONOFF_CLUSTER, 0x0000, [destEndpoint: 2])
```

Comment:
this check of `destEndpoint: 0xFF` was most likely important for the Orvibo device